integration-tests/interactions/user/invalid_input.jinja

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should this have a more precise name like `invalid_validation_code.jinja`?

Reply:
I used it as an invalid phone number too. Since specific validation should be tested in unit tests in most cases, I thought having a generic invalid input (a lonely space is incorrect in all but extremely weird contexts) would be practical instead of multiplying files with meaningless content.